mock-core-configs/etc/mock/templates/epel-8.tpl

```diff
@@ -73,14 +73,12 @@ gpgcheck=1
 gpgkey=file:///usr/share/distribution-gpg-keys/epel/RPM-GPG-KEY-EPEL-$releasever
 skip_if_unavailable=False
 
-{% if koji_primary_repo != None and koji_primary_repo != "epel" %}
-[local-epel]
-{% else %}
+{% if koji_primary_repo != None and koji_primary_repo == "epel" %}
 [local]
```

Comment:
> koji_primary_repo != None

What's the point of this? If the value is equal to "epel", it can't also be None.

Reply:
Good point.